mc-oblivious-ram/src/path_oram/mod.rs

```diff
@@ -251,6 +257,19 @@ where
         self.branch.checkin(&mut self.storage);
         debug_assert!(self.branch.leaf == 0);
 
+        for _ in 0..self.evictor.get_number_of_additional_branches_to_evict() {
+            let leaf = self.evictor.get_next_branch_to_evict();
+            debug_assert!(leaf != 0);
+            self.branch.checkout(&mut self.storage, leaf);
+            self.evictor.evict_from_stash_to_branch(
+                &mut self.stash_data,
+                &mut self.stash_meta,
+                &mut self.branch,
+            );
+            self.branch.checkin(&mut self.storage);
+            debug_assert!(self.branch.leaf == 0);
+        }
```

Comment:
:question: 
I'm probably missing it, but this looks like new behavior that wasn't there before, and yet not really seeing any testing of this behavior.
It looks like currently the evictor is created with 0 additional branches.  
Is this the pre-emptive logic to handle evicting both sides of the root for circuit oram?